controller_manager:
  ros__parameters:
    update_rate: 50

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    left_arm_velocity_controller:
      type: velocity_controllers/JointGroupVelocityController

    right_arm_velocity_controller:
      type: velocity_controllers/JointGroupVelocityController

    wheels_velocity_controller:
      type: velocity_controllers/JointGroupVelocityController

left_arm_velocity_controller:
  ros__parameters:
    joints:
      - base_link_left_arm_shoulder_joint
      - left_arm_1_left_arm_2_joint
      - left_arm_2_left_arm_3_joint

right_arm_velocity_controller:
  ros__parameters:
    joints:
      - base_link_right_arm_shoulder_joint
      - right_arm_1_right_arm_2_joint
      - right_arm_2_right_arm_3_joint

wheels_velocity_controller:
  ros__parameters:
    joints:
      - base_link_left_wheel_joint
      - base_link_right_wheel_joint
